assets/js/modules/adsense/components/settings/SettingsSetupIncomplete.js

```diff
@@ -41,9 +41,9 @@ export default function SettingsSetupIncomplete() {
 		<Fragment>
 			<AdBlockerWarning />
 
-			{ isPendingStatus && __( 'Site Kit has placed AdSense code on your site, ', 'google-site-kit' ) }
-			{ ! isPendingStatus && __( 'Setup incomplete: ', 'google-site-kit' ) }
-
+			{ isPendingStatus && __( 'Site Kit has placed AdSense code on your site:', 'google-site-kit' ) }
+			{ ! isPendingStatus && __( 'Setup incomplete:', 'google-site-kit' ) }
+			{ ' ' }
```

Comment:
This is concatenating here as it will either render

`Site Kit has placed AdSense code on your site:` + ` ` + [`check module page`](#)
or
`Setup incomplete:` + ` ` + [continue module setup](#)

Perhaps this would be a good use-case for [`createInterpolateElement`](https://make.wordpress.org/core/2020/07/17/introducing-createinterpolateelement/) ? That way the whole string with the markup could be translatable while allowing us to still use the `Link` component.

Reply:
I think we can keep this as is, IMO we should only need to use that if we need to break up a sentence into multiple pieces due to a component being used.